### PR TITLE
OE0-55: replace URL for IMIS CodeSystem

### PR DESCRIPTION
**Pull request by sniedzielski**
TICKET: https://openimis.atlassian.net/browse/OE0-55

### CODE REVIEW

File: api_fhir_r4/defaultConfig.py [hunk 1]
```diff
@@ -5,6 +5,7 @@
         "female": "F",
         "other": "O"
     },
+    "host_domain": "http://127.0.0.1:8000",
```
Comment (dragos-dobre) [inline api_fhir_r4/defaultConfig.py:8]: Try to get the host domain from "self.request.build_absolute_uri()". Thanks.

**Review by dragos-dobre — CHANGES_REQUESTED** [review]

File: api_fhir_r4/defaultConfig.py (hunk 1 above)
Comment (sniedzielski) [reply api_fhir_r4/defaultConfig.py:8]: ok

**Comment by sniedzielski** [comment]
@dragos-dobre @delcroip I used the approach proposed by @delcroip from that PR https://github.com/openimis/openimis-be_py/pull/53. On my computer tests passed when the change from main assembly module is applied. I also added full urls in references and I fixed bugs detected in some references.

**Comment by sniedzielski** [comment]
@delcroip thx for merging 'SITE_URL' in assembly module. @dragos-dobre tests now passed in that PR. Please review and later I will check on release server how this site_url works. Thx in advance

**Review by dragos-dobre — CHANGES_REQUESTED** [review]

File: api_fhir_r4/tests/test/test_group.json [hunk 2]
```diff
@@ -14,7 +14,7 @@
                {
                   "url":"https://openimis.github.io/openimis_fhir_r4_ig/StructureDefinition/address-location-reference",
                   "valueReference":{
-                     "reference":"Location/TEST_NAME-village"
+                     "reference":"/api/api_fhir_r4/Location/637f05cf-d8e8-4135-8250-7f01f01936bc"
```
[the remark this answers is not in the document]
Comment (sniedzielski) [reply api_fhir_r4/tests/test/test_group.json:17]: ok

[the remark this answers is not in the document]
Comment (sniedzielski) [reply api_fhir_r4/tests/test/test_group.json:17]: I will check this. I think I simply forgot remove this from fixtures/test payloads like here. But anyway I verify this if somewhere else we have such mistakes in references.

**Comment by sniedzielski** [comment]
@dragos-dobre I fixed what you requested. please review again, thx!

**Review by dragos-dobre — APPROVED** [review]

**Pull request merged**